src/test/scala/mesosphere/marathon/SchedulerActionsTest.scala

```diff
@@ -37,8 +37,7 @@ class SchedulerActionsTest extends AkkaUnitTest {
         .withAgentInfo(agentId = Some("slave 1"))
         .getInstance()
 
-      val instances = Seq(runningInstance, stagedInstance, stagedInstanceWithSlaveId)
-      f.instanceTracker.instancesBySpec() returns Future.successful(InstancesBySpec.forInstances(instances))
+      f.populateInstances(Seq(runningInstance, stagedInstance, stagedInstanceWithSlaveId))
```

Comment:
What's wrong with the original code?

Reply:
nothing, it just does not use this setup method...